zedd-app/src/components/AppBody.tsx

```diff
@@ -156,18 +244,7 @@ export const AppBody = observer(
         </div>
         <div>
           <div className={classes.controlBar}>
-            <TextField
-              label='Start'
-              type='date'
-              value={state.startDate}
-              onChange={(e) => (state.startDate = e.target.value)}
-            />
-            <TextField
-              label='End'
-              type='date'
-              value={state.endDate}
-              onChange={(e) => (state.endDate = e.target.value)}
-            />
+            {DateRangePicker(state)}
```

Comment:
This should be used as a regular react component, ie `<DateRangePicker value={state.showing} onChange={newValue => (state.showing=newValue)}/>`